test/userapi.py

```diff
@@ -22,9 +24,21 @@ def register_as_new(self):
     def login(self):
         return super().login(email=self.email)
 
-    def token(self):
-        response = requests.post(urljoin(self._baseurl, "/token"), headers=self._auth_header)
-        return self._check_response(response)
+    def token(self, access=None, set_token=False):
+        post_data = {}
+        if access is not None:
+            post_data["access"] = access
+
+        headers = self._auth_header.copy()
+        headers["Content-Type"] = "application/json"
+        response = requests.post(
+            urljoin(self._baseurl, "/token"), headers=headers, data=json.dumps(post_data)
```

Comment:
If you do `json=post_data` instead of converting it yourself and passing it via `data` then requests will also set  Content-Type to application/json for you.